client/state/selectors/is-site-vip.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Internal dependencies
+ */
+import { getRawSite } from 'state/sites/selectors';
+
+/**
+ * Returns true if the site is VIP
+ *
+ * If the site is missing returns null.
+ *
+ * @param  {Object}   state  Global state tree
+ * @param  {Number}   siteId Site ID
+ * @return {?Boolean}        Whether site is VIP
+ */
+export default function isSiteVip( state, siteId ) {
```

Comment:
Handled in https://github.com/Automattic/wp-calypso/pull/13400/commits/b74df8a3de71858f20cebc812a50051fd0a4b264.